spec/support/acts_as_tenant.rb

```diff
@@ -19,8 +18,32 @@ def with_tenant(tenant, &proc)
   end
 end
 
+module ActsAsTenant::RequestTestHelpers
+  include ActsAsTenant::TestGroupHelpers
+  # Sets the current tenant and host when running this group of tests.
+  #
+  # @param tenant [Symbol] The symbol containing the tenant to use for this group of
+  #                        tests. The tenant must have been set using a let construct.
+  # @param proc [Proc] The block containing the test definitions.
+  def with_tenant_and_host(tenant, &proc)
+    with_tenant(tenant) do
+      context "with host set" do |*params|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.